tests/cdc/run.sh

```diff
@@ -19,7 +20,17 @@ function prepare() {
     start_ts=$(cdc cli tso query --pd=http://$UP_PD_HOST:$UP_PD_PORT)
 
     run_cdc_server $WORK_DIR $CDC_BINARY
-    cdc cli changefeed create --start-ts=$start_ts --sink-uri="dsn://root@tcp(127.0.0.1:3306)/"
+
+    TOPIC_NAME="ticdc-cdc-test-$RANDOM"
+    case $SINK_TYPE in
+        mysql) SINK_URI="mysql://root@127.0.0.1:3306/";;
+        kafka) SINK_URI="kafka://127.0.0.1:9092/$TOPIC_NAME?partition-num=4";;
+        *) SINK_URI="mysql://root@127.0.0.1:3306/";;
+    esac
```

Comment:
```suggestion
    case $SINK_TYPE in
        kafka) SINK_URI="kafka://127.0.0.1:9092/$TOPIC_NAME?partition-num=4";;
        mysql) ;&  
        *) SINK_URI="mysql://root@127.0.0.1:3306/";;
    esac
```
we can save one line with the `fall-through` trick

Reply:
ok I'm not familiar with bash syntax